config/config_helpers_test.go

```diff
@@ -26,7 +26,7 @@ func TestPathRelativeToInclude(t *testing.T) {
 			"child",
 		},
 		{
-			&IncludeConfig{Path: "/root/.terragrunt"},
+			&IncludeConfig{Path: rootFolder + "/.terragrunt"},
```

Comment:
There are several other uses of `/root` in this file. Would you mind updating them the same way? That will help prevent these sorts of bugs in the future.